app/javascript/entrypoints/pdc/copy_to_clipboard.es6

```diff
@@ -0,0 +1,62 @@
+export default class CopytoClipboard {
+
+    constructor() {
+        // Setup the DOI's COPY button to copy the DOI URL to the clipboard
+        $('#copy-doi').click(this.copy_doi.bind(this));
+     }
+  
+    copy_doi() {
+        var doi = $('#copy-doi').data('url');
+        this.copyToClipboard(doi, '#copy-doi-icon', '#copy-doi-label', 'copy-doi-label-normal', 'copy-doi-label-copied');
+        return false;
+    }
+    
+    // Sets the elements to the proper CSS classes once a value has been copied to the clipboard.
+    setCopiedToClipboard(iconEl, labelEl, normalClass, copiedClass) {
+      $(iconEl).removeClass('bi-clipboard');
+      $(iconEl).addClass('bi-clipboard-check');
+      $(labelEl).text('COPIED');
+      $(labelEl).removeClass(normalClass);
+      $(labelEl).addClass(copiedClass);
```

Comment:
```suggestion
      $(iconEl).removeClass('bi-clipboard')
        .addClass('bi-clipboard-check');
      $(labelEl).text('COPIED')
        .removeClass(normalClass)
        .addClass(copiedClass);
```
method chaining is a jquery idiom.

Reply:
@mccalluc Can changing the code be a separate PR?  This PR was a direct copy to move it into Vite vs sprokets.  I would hate to look back and think these changes were needed to get Vite to compile the code.

Reply:
In addition I do not believe that it is good coding style to cram more than one action onto a line just because the code allows it.  Adding more than one action into a line makes it hard to understand what is happening when you read the code in the future.

Reply:
I won't block this, but I think there are cases where method chaining makes sense, but I'll try to summarize that in the meeting agenda.